docs/api.yaml

```
@@ -0,0 +1,1029 @@
+openapi: 3.0.3
+info:
+  title: p-log API
+  description: p-log application API documentation
+  version: 1.0.0
+servers:
+  - url: http://localhost:8080/api/v1
+    description: Local development server
+
+paths:
+  # Auth
+  /auth/oidc/login:
```

[must]oidcの設計違和感あって、今の状態だと

1. `/oidc/login`にアクセス
2. `/oidc/callback`にアクセス

となってるけど、普通リダイレクト処理はサーバー側で持つもんなのでちょっと認識違いあるかもしれない（あとcallbackに対してもPOSTじゃなくてGETなはず）

だから意図はわかるけど、普通に
```suggestion
  /auth/login:
```
だけにして、callbackはAPI定義書には書かないイメージ